helm/cluster/.helmignore

```diff
@@ -0,0 +1 @@
+README.md
```

Comment:
I added some README.md inside of templates, the idea is to have more detailed docs for many components (added an example for API server config), with parts being auto-generated.